### PR TITLE
Fix: Author & Version return values in alerts and Slack template

### PR DESCRIPTION
**Pull request by prakx87**
With Build Version Enabled:
image

Without Build Version Enabled:
image

### CODE REVIEW

File: services/tasks/alert.go [hunk 1]
```diff
@@ -382,7 +382,7 @@ func (t *TaskRunner) alertInfos() (string, string) {
 		author = user.Name
 	}
 
-	return version, author
+	return author, version
```
Comment (tboerger) [inline services/tasks/alert.go:385]: Good catch 🙈

File: services/tasks/templates/slack.tmpl [hunk 2]
```diff
@@ -13,7 +13,14 @@
                     "title": "Author",
                     "value": "{{ .Author }}",
                     "short": true
+                },
```
Comment (tboerger) [inline services/tasks/templates/slack.tmpl]: The comma should only be placed if a version is given

Comment (prakx87) [reply services/tasks/templates/slack.tmpl]: I have made changes as suggested and tested it.

image

**Comment by fiftin** [comment]
Thank you!

**Pull request merged**